test/avocado/selenium-sosreport.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
+
+# we need to be able to find and import seleniumlib, so add this directory
+import os
+import sys
+import time
+machine_test_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(1, machine_test_dir)
```

Comment:
since we probably need to do this for other files, I suggest we do something like this:
```
if not machine_test_dir in sys.path:
    sys.path.insert(1, machine_test_dir)
```
what do you think?